lib/00_external.sh

```diff
@@ -17,24 +18,27 @@ export _SUPPORTED_EXTERNALS="
 readonly _SUPPORTED_EXTERNALS
 
 _PACMAN_found_from_script_name() {
-  local _tmp_name=
-  local _pacman=
+  local_tmp_name=
+  local_pacman=
 
-  _tmp_name="${BASH_SOURCE[0]:-?}"
-  if [[ "$_tmp_name" == "?" ]]; then
-    _error "Unable to get script name."
+  local_tmp_name="${0}"
+  if echo "$local_tmp_name" | "$GREP" -q -- -; then
```

Comment:
Does this mean checking if "$local_tmp_name" contains `-`? Then:

```
case "$local_tmp_name" in
  *-*) : ;;
  *) return 1 ;;
esac

Reply:
yes this is to support non-system package manager (`pacapt-pip`, `pacapt-conda`,...) :D